types/pg/index.d.ts

```diff
@@ -44,11 +44,12 @@ export interface PoolConfig extends ClientConfig {
     // properties from module 'node-pool'
     max?: number | undefined;
     min?: number | undefined;
-    idleTimeoutMillis?: number | undefined;
+    idleTimeoutMillis?: number | undefined | null;
     log?: ((...messages: any[]) => void) | undefined;
     Promise?: PromiseConstructorLike | undefined;
     allowExitOnIdle?: boolean | undefined;
     maxUses?: number | undefined;
+    maxLifetimeSeconds?: number | undefined;
```

Comment:
https://github.com/brianc/node-postgres/pull/2698

I can split this out into a separate change, but it's relevant for `pool.options` so I included it here.